run_configure.mk

```diff
@@ -123,7 +123,7 @@ sh configure --disable-auto-build-flag 'OMRGLUE=$(OMRGLUE)' 'SPEC=$(SPEC)' $(CON
 touch $(CONFIGURE_OUTPUT_FILES)
 endef
 
-CONFIGURE_DEPENDENCIES := SPEC config.guess config.sub configure toolconfig/configure
+CONFIGURE_DEPENDENCIES := SPEC config.guess config.sub install-sh configure toolconfig/configure
```

Comment:
This was the result of a merge. I guess I misunderstood Robert's comment (read that he was adding dummy install-sh, instead he removed it). Reverting this to match Robert's commit (a7efc2a).